### PR TITLE
Update mdm-supported-devices.md

### PR DESCRIPTION
**Pull request by Reethan-Raj**

### CODE REVIEW

File: memdocs/intune/includes/mdm-supported-devices.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ ms.localizationpriority: high
 - Apple iPadOS 13.0 and later
 - macOS 10.15 and later
 
+> [!NOTE]
+> Intune requires iOS 13.x or higher for device enrollment scenarios and app configuration delivered through Managed devices app configuration policies.
```
Comment (JohanFreelancer9) [inline memdocs/intune/includes/mdm-supported-devices.md:18]:
```suggestion
> Intune requires iOS 13.x or later for device enrollment scenarios and app configuration delivered through Managed devices app configuration policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Reethan-Raj, kindly commit the suggested changes. Many thanks.

**Comment by ErikjeMS** [comment]
#sign-off

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@Reethan-Raj - thank you for committing my changes, but you seem to have missed one. Could you please commit that as well? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**